src/components/loading-indicators/loading-members-list.component.js

```diff
@@ -47,20 +49,20 @@ export class LoadingMembersList extends Component {
     const { title } = this.props;
 
     return (
-      <View style={styles.wrapper}>
-        <Text style={styles.sectionTitle}>{title}</Text>
+      <Wrapper>
+        <SectionTitle>{title}</SectionTitle>
 
-        <View style={styles.avatarContainer}>
+        <AvatarContainer>
           {[...Array(10)].map((item, index) => {
             return (
-              <Animated.View
+              <Avatar
                 key={index} // eslint-disable-line react/no-array-index-key
-                style={[styles.avatar, { opacity: this.state.fadeAnimValue }]}
+                style={{ opacity: this.state.fadeAnimValue }}
```

Comment:
How about designing `opacity` as a prop of `Avatar`? Then,
```
const Avatar = styled(Animated.View)`
    opacity: ${prop => prop.opacity};
    // blabla
`
```

Reply:
@chinesedfan  I pass the prop from the parent component 
`<LoadingMembersList title={t('MEMBERS', locale)} opacity={new Animated.Value(0)}/>`

I use it in Avatar using props
`opacity: ${props => props.opacity};`

I'm able to get the props in the constructor and it is of type _AnimatedValue_

But inside the styled component the props seems to be undefined and hence I get the below exception in the device. Any idea why?

![simulator screen shot 26-oct-2018 1 03 26 pm](https://user-images.githubusercontent.com/6077433/47551407-bc3b0c00-d91f-11e8-9717-2c5deefc2465.png)

Reply:
@chinesedfan any other changes required here?

Reply:
The space between section CONTRIBUTORS and section SOURCE are larger before. Is it normal?

Reply:
I tried reverting to master and took a screenshot again. 
![before](https://user-images.githubusercontent.com/6077433/47568535-fa9bef80-d94e-11e8-824d-e80c9aa692f8.png)

Not sure what was causing the extra space in the first place. The spacing right now is what is present in the master.

Reply:
At first, 

- with master codes, it was always larger space
- with your codes, it was smaller space at the first render. And if visited the second repository, it became larger space

Then I removed all `node_modules` and `yarn` again. It is always larger space, no matter with master or your codes. Well, it's no a big problem, so don't mind about it.